examples/console/main.go

```diff
@@ -381,12 +382,14 @@ func prompt(agent *bufferedagent.Agent, stats *stats, submitter agentpkg.Submitt
 		if err != nil {
 			return err
 		}
+		rand := rand.New(rand.NewSource(time.Now().UnixNano()))
 		stats.Reset()
 		agent.SetMaxBufferSize(bufferSize)
 		stats.MarkStart()
 		for i := 0; i < x; i++ {
 			memo := "tx-" + strconv.Itoa(i)
 			for {
+				amt := rand.Int63n(amt) + 1
```

Comment:
❓ will this hit the limit of the amount we deposit? i thought that's why you would use a very small value less than 1 during the demos

Reply:
If you run a test with an amount of 0.01, it will find random values between 0.0000001 and 0.01.